github/event_types.go

```diff
@@ -490,12 +490,14 @@ type PullRequestEvent struct {
 	PullRequest *PullRequest `json:"pull_request,omitempty"`
 
 	// The following fields are only populated by Webhook events.
-	Changes            *EditChange   `json:"changes,omitempty"`
-	RequestedReviewers []*User       `json:"requested_reviewers,omitempty"` // Populated in "review_requested", "review_request_removed" event deliveries.
-	Repo               *Repository   `json:"repository,omitempty"`
-	Sender             *User         `json:"sender,omitempty"`
-	Installation       *Installation `json:"installation,omitempty"`
-	Label              *Label        `json:"label,omitempty"` // Populated in "labeled" event deliveries.
+	Changes *EditChange `json:"changes,omitempty"`
+	// RequestedReviewer is populated in "review_requested", "review_request_removed" event deliveries. A request for
+	// multiple users to review at once is split into multiple requests, each with a single, different RequestedReviewer.
```

Comment:
I think it might be more clear to write that:

```diff
-A request for multiple users to review at once
-is split into multiple requests, each with ...
+A request affecting multiple reviewers at once
+is split into multiple "review_requested"/"review_request_removed" event deliveries, each with ...
```

They're split into multiple `PullRequestEvent`s, not multiple requests.

Reply:
@shurcooL @gmlewis Agreed, pushed in this comment change